scripts/replace-cli-version.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+VERSIONS=$(gh api -H "Accept: application/vnd.github+json" /repos/github/codeql-cli-binaries/releases | jq '.[].tag_name' | head -2)
+
+LATEST_VERSION=$(echo $VERSIONS | awk '{ print $1 }' | sed "s/\"//g")
+PREVIOUS_VERSION=$(echo $VERSIONS | awk '{ print $2 }' | sed "s/\"//g")
```

Comment:
With the `-r` option above:

```suggestion
LATEST_VERSION=$(echo $VERSIONS | awk '{ print $1 }')
PREVIOUS_VERSION=$(echo $VERSIONS | awk '{ print $2 }')
```

Reply:
Also, see above...can you export these variables so they can be used in the PR title and body?